starknet_devnet/blocks.py

```diff
@@ -32,7 +32,13 @@ def _parse_block_hash(raw: Optional[str]):
         return raw
 
     try:
-        return int(raw, 16)
+        if raw.startswith("0x"):
+            return int(raw, 16)
+
+        raise StarknetDevnetException(
+            code=StarkErrorCode.MALFORMED_REQUEST,
+            message=f"Block hash should be a hexadecimal string starting with 0x, or 'null'; got: '{raw}'.",
```

Comment:
Is this the message sent by the official feeder gateway? And is `MALFORMED_REQUEST` what's being sent by the gateway?